src/olympia/api/serializers.py

```diff
@@ -63,8 +63,7 @@ def handle_date(self, value):
         # Don't be picky about microseconds here. We get them some time
         # so we have to support them. So let's strip microseconds and handle
         # the datetime in a unified way.
-        value = value.partition('.')[0]
-        return datetime.strptime(value, '%Y-%m-%dT%H:%M:%S')
+        return datetime.fromisoformat(value.partition('.')[0])
```

Comment:
That we're stripping off microseconds makes me wonder - _what_ format exactly does ES guarantee to return it's datetimes in?  (And is that format always something that `fromisoformat` will parse?)

Reply:
The [docs](https://www.elastic.co/guide/en/elasticsearch/reference/7.1/date.html) specify how to deal with the input a lot more than output... it just says
> Dates will always be rendered as strings, even if they were initially supplied as a long in the JSON document.

As far as I can see by making requests, it's always in ISO 8601 with the full time including microseconds... but that precision may be lost by MySQL. Stripping them is an attempt to keep the 2 consistent IIRC.